src/Datadog.Trace/DiagnosticListeners/AspNetCoreDiagnosticObserver.cs

```diff
@@ -289,6 +289,7 @@ private void OnHostingHttpRequestInStart(object arg)
 
             scope.Span.DecorateWebServerSpan(resourceName, httpMethod, host, url, tags, tagsFromHeaders);
 
+            tags.Measured = 1;
```

Comment:
Looks like we already add the metric in `span.DecorateWebServerSpan(...)` above, so we don't need to add it again here.